### ISSUE TITLE
[BUG] Hydra as a dependency to multicursors throws warning

### ISSUE THREAD

**Issue by saygo-png** [nix-community/nixvim#1943]
| Field        | Description |
|--------------|-------------|
| Plugin       | `multicursors` `hydra`  | 
| Nixpkgs      | `24.05` | 
| Home Manager | `24.05` | 

- [x] I have read the [FAQ](https://nix-community.github.io/nixvim/user-guide/faq.html) and my bug is not listed there.

## Description
Launching nvim greets you with a warning
[Hydra.nvim] Option "hint.border" has been deprecated and will be removed on 2024-02-01 -- See hint.float_opts

## Minimal, Reproducible Example (MRE)

```nix
      programs.nixvim = {
        enable = true;
        plugins = {
          multicursors = {
            enable = true;
          };
        };
      };
```

Temporary fix:
```nix
extraConfigLuaPre = ''
  -- Hide depreciation warnings
  vim.notify = function(msg, ...)
    if msg:match("has been deprecated") then
      return
      end
    notify(msg, ...)
  end
'';
```

**Comment by lovesegfault** [comment]
I think this is fixed now that `nixpkgs` moved to the hydra.nvim fork:

https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/editors/vim/plugins/generated.nix#L4824-L4834

https://github.com/nvimtools/hydra.nvim

**Comment by GaetanLepage** [comment]
@saygo-png is this indeed fixed ?

**Comment by saygo-png** [comment]
No, as far as i can tell both master and stable use the same hydra.nvim, which gives the warning, and its the same version now as it was  when i made the issue. I am using stable nixos but i looked at the different nixpkgs branches too see which versions they were using, and they pull from the same hash
```nix
  inputs = {
    nixpkgs.url = "github:nixos/nixpkgs/nixos-24.05";
    home-manager = {
      url = "github:nix-community/home-manager/release-24.05";
      inputs.nixpkgs.follows = "nixpkgs";
    nixvim = {
      url = "github:nix-community/nixvim/nixos-24.05";
      inputs.nixpkgs.follows = "nixpkgs";
      inputs.home-manager.follows = "home-manager";
      inputs.nix-darwin.follows = ""; # no apple here
    };
  };
```

**Comment by GaetanLepage** [comment]
> Fixed in 24.11

Good to hear, thanks for reporting back.
